Sources/Charcoal/Selection/FilterSelectionStore.swift

```diff
@@ -140,7 +140,8 @@ extension FilterSelectionStore {
                 return ["\(lowValue ?? "...") - \(highValue ?? "...")"]
             }
         case .stepper:
-            return value(for: filter).map({ [$0] }) ?? []
+            guard let lowValue: Int = value(for: filter) else { return [] }
+            return ["\(lowValue)+"]
```

Comment:
`return value(for: filter).map({ ["\($0)+"] }) ?? []` should also work, but keep `guard` if you like it more 😉

Reply:
I changed if `if let` instead to be more consistent with the other cases. That map thing is a nice one liner though 😄